extensions/wikia/PageShare/PageShareHelper.class.php

```diff
@@ -17,6 +17,25 @@ public static function getIcon( $service ) {
 		return $icon;
 	}
 
+	/**
+	 * Get language for Page Share service.
+	 * For Anon users use the browser language.
+	 * For logged in user use user's language.
+	 * Both values can be overwritten by ?uselang parameter.
+	 *
+	 * @return String language
+	 */
+	public static function getLangForPageShare() {
+		global $wgLang, $wgRequest, $wgUser;
+
+		if ( $wgUser->isAnon() ) {
+			$lang = substr($_SERVER['HTTP_ACCEPT_LANGUAGE'], 0, 2);
```

Comment:
As far as I know we don't have such list. For page share we only need the two-chars language's code so even if we have such list we would have to take substr from the result. Right now Page Share displays all of social networks and includes/excludes some base on language.